npc/re/mobs/dungeons/odin.txt

```diff
@@ -54,3 +51,17 @@ odin_tem03,0,0	monster	Valkyrie	1765,2,5400000,1800000
 odin_tem03,0,0,0,0	boss_monster	Valkyrie Randgris	1751,1,28800000,600000,1
 odin_tem03,0,0	monster	Blue Plant	1079,18,1800000,900000
 odin_tem03,0,0	monster	Shining Plant	1083,18,1800000,900000
+
+//==================================================
+// odin_past - Ancient Odin Shrine
+//==================================================
+odin_past,0,0	monster	Angelgolt	20604,40,5000
+odin_past,0,0	monster	Angelgolt	20605,60,5000
+odin_past,0,0	monster	Holy Frus	20606, 45,5000
+odin_past,0,0	monster	Holy Skogul	20607, 45,5000
+odin_past,0,0	monster	Arch Plasma	20608,5,5000
+odin_past,0,0	monster	Spectral Plasma	20609,5,5000
+odin_past,0,0,0,0	boss_monster	Valkyrie Reginleif	20610,1,28800000,600000,1
```

Comment:
The mpvs Reginleif and Ingrid aren't they randomly summoned after killing a certain amount of Holy Frus and Holy Skogul?

Reply:
**Valkyrie Ingrid** appears after the entire server has eliminated 300 Holy Skogul. The recount is done every 6 hours.
**Valkyrie Reginleif** appears after the entire server has eliminated 300 Holy Frus. The recount is done every 6 hours.

official information from bRO. - https://browiki.org/wiki/MVP#cite_ref-6